README.md

```diff
@@ -69,8 +69,6 @@ The dependencies for building .NET from source can be found [here](https://githu
     cd /path/to/complete/dotnet/sources
     ./prep.sh
     ```
-    
-    On arm64, please use `./prep.sh --bootstrap` instead. This issue is being tracked [here](https://github.com/dotnet/source-build/issues/2758).
```

Comment:
This step should be preserved.  The second sentence regarding the tracking issue can be removed.  You will always have to --bootstrap on non x64 architecture when building for the first time.

Reply:
Reverted